test_suite/conftest.py

```diff
@@ -32,15 +39,36 @@ def check_markers(host, request):
     ]
 
     host_distro = host.system_info.distribution
-    host_version = host.system_info.release
+    host_version = float(host.system_info.release)
     distro_version = f'{host_distro}{host_version}'
 
     # Skip the test if the distro is explicitly excluded
     if exclude_on_marker:
         exclude_on_marker_list = exclude_on_marker.args[0]
         if host_distro in exclude_on_marker_list \
                 or distro_version in exclude_on_marker_list:
-            pytest.skip(f"This test doesn't apply to {distro_version}")
+            pytest.skip(skip_massage.format(distro_version=distro_version))
+
+        elif len(exclude_on_marker_list) > 1 and host_distro in exclude_on_marker_list[0]:
```

Comment:
Why are we checking if this list contains more than one item? Does this mean that it won'r work if we pass only one item? E.g. '>rhel9.0'

Reply:
I forgot to mention how to use this, so here how it supposed to be used:
Until now we used it like this:
@pytest.mark.exclude_on(['rhel7.9'])
@pytest.mark.exclude_on(['rhel9'])
and so on...

With the fix you can still use the previous way to exclude a specific version or to set a sign '<' or '>' to refer the version you put in the fixrt index of the string.
@pytest.mark.exclude_on(['rhel7.9', '<'])
@pytest.mark.exclude_on(['rhel8.5', '>'])
I f that doesn't make sense to you we can think of something else together

Reply:
Oh, thanks. Now I understand by you check if there is more than 1 item in the list! I would invert the order anyways, so it is more readable and it looks more natural.
E.g. `@pytest.mark.exclude_on(['<', 'rhel8.8'])` (so, all versions lower than RHEL 8.8 will be skipped).